jest.config.js

```diff
@@ -17,9 +17,6 @@ const customJestConfig = {
   modulePathIgnorePatterns: ['/\\.next/'],
   modulePaths: ['<rootDir>/lib'],
   transformIgnorePatterns: ['/next[/\\\\]dist/', '/\\.next/'],
-  globals: {
-    AbortSignal: global.AbortSignal,
-  },
```

Comment:
We can get rid of this thanks to https://jestjs.io/blog/2022/04/25/jest-28#all-nodejs-globals